fastlane/jetpack_metadata/zh-Hans/release_notes.txt

```diff
@@ -0,0 +1,5 @@
+We don’t know about you, but we’re feeling 22. In honor of the occasion (and because you’ve been asking for it), you can now transform most types of blocks into other block types, like quotes, columns, and groups. 不错！
+
+We made some design adjustments to the Reader, updating the “followed topics/sites” icon, recoloring the navigation bar background to match other screens in the app, and resizing the too-large title in the Reader tab. (We did that last one in the Notifications tab, too.)
+
+You’ll also see a few small visual changes to the My Site tab bar icon and the in-app survey. And if you have an extra-sharp eye, you might notice the dark background color of toolbars and top tabs is now consistent across the app.
```

Comment:
This has been flagged internally, pxLjZ-7Kx-p2, and so has the WordPress counterpart.

I did not consider it a blocker to uploading the metadata and submitting the apps for review. They'll be alive in the store for ~14 days then be replaced.